FirebaseCombineSwift/Tests/Unit/Bundle+Tests.swift

```diff
@@ -0,0 +1,24 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+extension Bundle {
+  class func mock(with cls: Bundle.Type) {
```

Comment:
Can you elaborate a bit on the functionality of this utility?

Reply:
`NSBundle` is used under the hood on `OAuthProvider` to access values such as bundleIdentifier and `object(forInfoDictionaryKey:)` which cannot be injected using internal APIs. Instead, I introduced a utility method to mock the `Bundle` with a subclass to full fill the tests expectations.